recipe/build.sh

```diff
@@ -73,3 +75,11 @@ cmake --install .
 
 # Clean up after build
 rm -rf "${PREFIX}/lib/libc++.tbd"
+
+if [[ "$target_platform" == "$build_platform" ]]; then
+  RESOURCE_DIR=$(${PREFIX}/bin/clang -print-resource-dir)
+  if [[ "${RESOURCE_DIR}" != "${INSTALL_PREFIX}" ]]; then
+    echo "Wrong install prefix (${INSTALL_PREFIX}). Should match ${RESOURCE_DIR}"
+    exit 1
+  fi
+fi
```

Comment:
Would it make sense to add such a test on windows as well?

Reply:
Too lazy to write it in batch script. PRs welcome.